src/main/java/com/blackducksoftware/integration/hub/jenkins/PostBuildHubScan.java

```diff
@@ -199,6 +202,15 @@ public boolean perform(final AbstractBuild<?, ?> build, final Launcher launcher,
         return true;
     }
 
+    private boolean isFailureConditionsConfigured(final AbstractBuild<?, ?> build) {
+        for (final Publisher publisher : ((FreeStyleProject) build.getParent()).getPublishersList()) {
+            if (publisher instanceof HubFailureConditionStep) {
```

Comment:
I think that because it's simple enough to get the value from the checkbox, it's better to potentially save minutes of build time -- especially since the build will go unstable regardless (since failure conditions are not configured) -- than to always add scan time if that post build action exists.